opencraft/settings.py

```diff
@@ -475,3 +475,15 @@
 # A new relic admin user's API key, used to set up availability monitoring
 # with Synthetics
 NEWRELIC_ADMIN_USER_API_KEY = env('NEWRELIC_ADMIN_USER_API_KEY', default=None)
+
+
+# Load balancing ##############################################################
+
+# The load-balancing server given in the form ssh_username@server.domain will be created
+# in the database if it does not exist yet.
+DEFAULT_LOAD_BALANCING_SERVER = env('DEFAULT_LOAD_BALANCING_SERVER', default=None)
+
+LOAD_BALANCER_FRAGMENT_NAME_PREFIX = env('LOAD_BALANCER_FRAGMENT_NAME_PREFIX', default='opencraft-')
+LOAD_BALANCER_CONF_DIR = env('LOAD_BALANCER_CONF_DIR', default='/etc/haproxy/conf.d')
+LOAD_BALANCER_BACKENDS_DIR = env('LOAD_BALANCER_BACKENDS_DIR', default='/etc/haproxy/backends')
```

Comment:
Yep, the paths are a bit out of place here.  I thought about introducing commands like `add-haproxy-conf-fragment` and `add-haproxy-backend-map-fragment` on the load balancer, and make these commands the public interface of the load balancer.  That would also allow to do the `flock` locking on the load balancer (meant to avoid reading half-written fragments) completely transparent.